VideoClassification/test.py

```diff
@@ -0,0 +1,99 @@
+"""Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Defines the testing procedure.
+"""
+import os
+import time
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import tensorflow.keras.metrics as metrics
+import numpy as np
+
+import readers
+import reader_utils
+import model as model_lib
+import loss
+
+def test_model(model, data_reader, test_dir, batch_size):
+	"""Test the model on test dataset attained from test_dir.
+	
+	Args:
+		model: the trained keras model
+		data_reader: the TfRecords datareader used to load training and validation datasets
+		test_dir: string representing the directory containing the test TfRecords files
+	Returns:
+		eval_dict: dictionary containing important evaulation metrics
+	"""
+	#Create evaluation metrics
+	auc_calculator = metrics.AUC(multi_label=True)#, curve='PR')
```

Comment:
Actually we want the curve='PR', right?

Reply:
yes, I added this just for testing purposes. I did not realize this would push to the merge.